moodle/Tests/MoodleCSBaseTestCase.php

```diff
@@ -146,9 +146,6 @@ protected function setSniff($sniff) {
      * @param string $fixture full path to the file used as input (fixture).
      */
     protected function setFixture($fixture) {
-        if ($this->fixtureContent !== null) {
```

Comment:
Okay, so the main reason I wanted it was to be able to mock filepaths without having to duplicate the entire thing on disk.

The phpcs DummyFile accepts this syntax at the top of the file to do so:
```php
phpcs_input_file: [file/path/to/use]
<?php
// ...   
```

Rather than having this duality (which I agree is crap) I've instead provided an optional argument to `setFixture` for the filename to use. If specified then we source the fixture and prepend the input_file, passing it into a DummyFile object instead.

Reply:
> ... I've instead provided an optional argument to setFixture for...

Still, should we reset it in the `tearDown()` method, like we do for everything else in the base class?

Heh, note that exactly that was my main point. In any case, glad with the change to fixture + path.

Reply:
Sorted - also added the missing `$this->fixture`